docs/reference/src/advanced/backends_and_renderers.md

```diff
@@ -80,3 +80,20 @@ The Qt renderer comes with the [Qt backend](backend_qt.md) and renders using QPa
  - Available in the [Winit backend](backend_winit.md) and [LinuxKMS backend](backend_linuxkms.md).
  - Public [C++](slint-cpp:api/classslint_1_1platform_1_1SkiaRenderer) API.
 
+#### Troubleshooting
+
+You may run into compile issues when enabling the Skia renderer. The following sections track
+issues we're aware of and how to resolve them.
+
+* Compilation error on Windows with messages about multiple source files and unused linker input
+
+  You may see compile errors that contain this error and warning from clang-cl:
+  ```
+   clang-cl: error: cannot specify '/Foobj/src/fonts/fontmgr_win.SkFontMgr_indirect.obj' whencompiling multiple source files
```

Comment:
> whencompiling

Is the space missing in the original error message?

Reply:
Ah no, it's in the original message. I'll add it. Well spotted.